views/feed.ejs

```diff
@@ -13,6 +13,7 @@
             <description><%= striptags(post.content) %></description>
             <pubDate><%= moment.utc(post.timestamp).format("ddd, DD MMM YYYY HH:mm:ss ZZ") %></pubDate>
             <category><%= post.categories[0] %></category>
+			<link>https://blog.replit.com/<%= post.url %></link>
```

Comment:
not sure why the indentation looks weird - is it just GitHub?

Reply:
Yeah I believe so, it looked normal to me on Replit.